apps/web/src/pages/integrations/constants.ts

```diff
@@ -0,0 +1,6 @@
+export enum IntegrationsStoreModalAnalytics {
+  CREATE_INTEGRATION_FORM_SUBMIT = '[Integrations Store Modal] Create Integration Form Submit',
```

Comment:
```suggestion
  CREATE_INTEGRATION_FORM_SUBMIT = 'Create Integration Form Submit - [Integrations Store Modal]',
```

A small suggestion is to move the category to the end. It's just easier to read on mixpanel because their events dropdown are quite small, so you would only read the repeated beginning :)